app/controller/auth.js

```diff
@@ -52,42 +56,52 @@ app.get('/logout', function(req, res) {
   res.redirect('/');
 });
 
-// Export some convienience functions
+// Convienience functions for authentication in express
 export function loggedIn(req, res, next) {
   if (req.isAuthenticated()) {
     return next();
   }
+
   return res.redirect('/auth/github');
 }
 
 export function isBeta(req, res, next) {
```

Comment:
I still really think these functions could be abstracted very nicely into a single higher-level `hasRole(r)` function, which would take a string (or even better, some symbolic/enum type derived from the Mongoose model...) and return a middleware function.

Might be something worth doing in this refactor, what do you think?